server/cp/ingest/parser/cp_onclusive.py

```diff
@@ -46,16 +48,21 @@ def parse(self, content, provider=None):
                                     }
                                 )
                     if subject["scheme"] == "onclusive_event_types":
-                        for event_item in event_types_cvs:
-                            if event_item["name"] == subject["name"]:
+                        cp_event_type = self.find_cv_item(
+                            cp_event_types, subject["name"].lower()
+                        )
+                        if cp_event_type and cp_event_type.get("is_active"):
```

Comment:
imo would be good to do that `is_active` filtering in the `_get_cv_items` method as we should only consider active items everywhere